doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -79,82 +79,80 @@ Flags, Options and Tables
 -----------------------------
 
 Coq has many settings to control its behavior.  Setting types include flags, options
-and tables.  Flags are boolean values, such as :flag:`Printing All`.  Options have numeric or
-string values, such as :opt:`Printing Width`.  Tables contain sets of strings.
-(In addition, some commands just provide settings, for example :cmd:`Extraction Language OCaml`.)
+and tables:
+
+* A :production:`flag` has a boolean value, such as :flag:`Asymmetric Patterns`.
+* An :production:`option` has a numeric or string value, such as :opt:`Firstorder Depth`.
+* A :production:`table` contains a set of strings.
+* In addition, some commands provide settings, such as :cmd:`Extraction Language OCaml`.
+
+.. FIXME Convert `Extraction Language OCaml` to an option.
 
 Flags, options and tables are identified by a series of identifiers, each with an initial
 capital letter.
 
-.. TODO : flag is not a syntax entry
-
 .. cmd::  {? Local | Global | Export } Set @flag
    :name: Set
```

Comment:
I'm not sure I understand what you said: did you try putting `:name: Set @flag` here?